pandas/core/base.py

```diff
@@ -1199,7 +1210,7 @@ def factorize(self, sort=False, na_sentinel=-1):
         dtype: int64
 
         >>> x.searchsorted(4)
-        array([3])
+        array([3])  # Note: an array, not a scalar
```

Comment:
Will this cause a doctest to fail?

Reply:
It does not fail, but is probably not needed. I'll just remove that comment.